static/js/src/base/navigation.js

```diff
@@ -88,3 +88,66 @@ document.addEventListener("click", function (event) {
 });
 
 export { toggleSubnav };
+
+var origin = window.location.href;
```

Comment:
@steverydz do you think it could go on it own file?

Reply:
@tbille yes, I think that would make sense. It could be `static/js/src/publisher/analytics.js`

Reply:
As an FYI, ad blockers are actively blocking JS files named "analytics.js", "GA.js", etc.

Reply:
What about `/publisher/navigation-events.js`?

Reply:
`navigation-events.js` sounds good. The path should be `static/js/src/public/`, my mistake.